x11-themes/terminology-themes/metadata.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+<maintainer type="person">
+	<email>cyrevolt@googlemail.com</email>
+	<name>Daniel Maslowski</name>
+</maintainer>
```

Comment:
Please also include proxy-maint team (copy it off some other metadata.xml), so that we get CC-ed on bugs.